rules/windows/process_creation/win_susp_procdump.yml

```diff
@@ -19,14 +19,14 @@ logsource:
     product: windows
 detection:
     selection1:
-        CommandLine:
-            - '* -ma *'
+        CommandLine|contains:
+            - ' -ma '
     selection2:
-        CommandLine:
-            - '* lsass*'
+        CommandLine|contains:
+            - ' lsass'
     selection3:
-        CommandLine:
-            - '* -ma ls*'
+        CommandLine|contains:
+            - ' -ma ls'
```

Comment:
could you please use the following approach here:

```
        CommandLine|contains|all:
            - ' -ma'
            - 'ls'
```

Reply:
Applied